packages/ember-intl/package.json

```diff
@@ -35,34 +35,34 @@
     "test": "ember test --test-port=0"
   },
   "dependencies": {
-    "@babel/core": "^7.25.8",
-    "@formatjs/icu-messageformat-parser": "^2.7.10",
-    "@formatjs/intl": "^2.10.8",
+    "@babel/core": "^7.26.0",
+    "@formatjs/icu-messageformat-parser": "^2.9.7",
+    "@formatjs/intl": "^3.0.4",
```

Comment:
`@formatjs/intl@3.0.0` removes a couple of dependencies.

https://github.com/formatjs/formatjs/blob/%40formatjs/intl%403.0.4/packages/intl/CHANGELOG.md